apps/settings/locales/settings.en-US.properties

```diff
@@ -1130,8 +1130,9 @@ draw-tile-borders=Draw tile borders
 draw-layer-borders=Draw layer borders
 log-animations=Log slow animations
 remote-debugging-usb=Debugging via USB
-dev-tools-wifi=DevTools via Wi-Fi
-dev-tools-qps=Pseudo-localization
+devtools-wifi=DevTools via Wi-Fi
+devtools-device-name=Device Name
```

Comment:
This file localizes many different screens in settings, so let's look at the developer menu in particular. Judging from the settings there, the consensus appears to be "Sentence case" (caps only for first letter, letter after punctuation, acronyms like "Wi-Fi" or "ADB" and proper nouns like "Marketplace"). There are only two offenders right now: "Cards View: Screenshots" and "Update Channel".

There have been many back and forths in the past. If you feel like it, you could change the whole menu back to "Title Case" (as the [UX spec](https://bugzilla.mozilla.org/show_bug.cgi?id=995124) originally wanted), but I'll let you figure out what to do with smaller works like in "Show screen reader settings", "Shake to save system log" or "Geolocation output in ADB".